### DISCUSSION TITLE
No formatter avaliable

### DISCUSSION THREAD

**Discussion by Set27** [LazyVim/LazyVim#3096] [Q&A] [answered]
Hi. I install first formatter that support JS, then I try to format it (space + c + f) and see that "No formatter available". What I am missing?
![image](https://github.com/LazyVim/LazyVim/assets/49077110/13adf9bc-49ed-46ee-9964-20e9ef3d226e)
![image](https://github.com/LazyVim/LazyVim/assets/49077110/91482283-b591-4e0f-9c19-a9ed7b5da18b)

**Comment by dpetka2001** [comment, marked as answer]
You also have to configure `conform.nvim` to use the formatter you want for the filetype you want. See [docs](https://www.lazyvim.org/plugins/formatting). Specifically you must add it in
```lua
formatters_by_ft = {
lua = { "stylua" },
fish = { "fish_indent" },
sh = { "shfmt" },
},
```
